JavaInterop/kotlin-lib/src/main/java/org/example/Person.kt

```diff
@@ -0,0 +1,38 @@
+package org.example
+
+import io.realm.kotlin.ext.realmListOf
+import io.realm.kotlin.types.RealmList
+import io.realm.kotlin.types.RealmObject
+import io.realm.kotlin.types.annotations.PersistedName
+import io.realm.kotlin.types.annotations.PrimaryKey
+import org.mongodb.kbson.ObjectId
+
+class Person: RealmObject {
+    @PrimaryKey
+    @PersistedName("_id")
+    var id = ObjectId()
+    var name: String = ""
+    var age: Int = 0
+    var children: RealmList<Child> = realmListOf()
+    var favoriteChild: Child? = null
+
+    // Expose method making it easier to use from Java
+    fun setChildren(children: ArrayList<Child>) {
```

Comment:
If named `set` then we should probably clear the underlying list before adding `children`.